mkat_fpga_tests/test_cbf.py

```diff
@@ -1065,3 +1085,210 @@ def disable_fft_overflow():
                      .format(flag_descr, condition))
         Aqf.equals(other_set_bits3, set(), 'Check that no other flag bits (any of {}) '
                      'are set.'.format(sorted(other_bits)))
+
+    def _delay_phase(self, delay_phase_dict, setup_data):
```

Comment:
I think this test is aggregated at too high a level. Rather split out  `get_actual_fringes()` as a helper method and then implement the high-level logic of each `?delays` related test separately, using the helper methods to cut down on code duplication.